client_request_metrics_local_remote_test.py

```diff
@@ -59,7 +59,7 @@ def test_batch_and_slice(self):
         # Run batch test:
         query = 'BEGIN BATCH '
         for i in murmur3_hashes.keys():
-            for y in range(0, 50):
+            for y in range(0, 40):
```

Comment:
Why this change?

Reply:
Iirc I was getting 'batch too large` errors but I reverted and they don't seem to show up anymore